src/scripts/lexmatch-sssom-compare.py

```diff
@@ -159,134 +144,49 @@ def extract_unmapped_matches(matches: TextIO, output_dir: str, summary: TextIO):
-    )
-    unmapped_gard_df = get_unmapped_df(
-        gard_comparison_df, in_lex_but_not_mondo_list, in_mondo_but_not_lex_list
-    )
-    summary.write("## unmapped_xxxx_lex & unmapped_xxxx_lex_exact")
-    summary.write("\n")
-
-    export_unmatched_exact(
-        unmapped_icd_df, "LEXMATCH", join(LEXMATCH_DIR, "unmapped_icd_lex.tsv"), summary
-    )
-    export_unmatched_exact(
-        unmapped_omim_df,
-        "LEXMATCH",
-        join(LEXMATCH_DIR, "unmapped_omim_lex.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_ordo_df,
-        "LEXMATCH",
-        join(LEXMATCH_DIR, "unmapped_ordo_lex.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_doid_df,
-        "LEXMATCH",
-        join(LEXMATCH_DIR, "unmapped_doid_lex.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_ncit_df,
-        "LEXMATCH",
-        join(LEXMATCH_DIR, "unmapped_ncit_lex.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_gard_df,
-        "LEXMATCH",
-        join(LEXMATCH_DIR, "unmapped_gard_lex.tsv"),
-        summary,
-    )
-
-    summary.write("## unmapped_xxxx_mondo")
-    summary.write("\n")
-
-    export_unmatched_exact(
-        unmapped_icd_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_icd_mondo.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_omim_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_omim_mondo.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_ordo_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_ordo_mondo.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_doid_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_doid_mondo.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_ncit_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_ncit_mondo.tsv"),
-        summary,
-    )
-    export_unmatched_exact(
-        unmapped_gard_df,
-        "MONDO_MAPPINGS",
-        join(mondo_match_dir, "unmapped_gard_mondo.tsv"),
-        summary,
-    )
-
-    combined_df = pd.concat(
-        [
-            unmapped_icd_df,
-            unmapped_omim_df,
-            unmapped_ordo_df,
-            unmapped_doid_df,
-            unmapped_ncit_df,
-            unmapped_gard_df,
-        ]
-    )
+    condition_lex_df_mondo_subj = lex_df["subject_id"].str.contains("MONDO")
+    ont_df_list = []
+
+    for idx, ont in enumerate(input):
+        ont2 = ont.upper()
+        if ont == "omim":
+            ont2 = "|".join((["OMIM", "OMIMPS"]))
+
+        mondo_ont_df = msdf_mondo.df[condition_mondo_sssom_subj & msdf_mondo.df['object_id'].str.contains(ont2)]
+        mondo_ont_lex_df = lex_df[(condition_lex_df_mondo_subj & lex_df['object_id'].str.contains(ont2))]
```

Comment:
`contains` deals with this OMIM|OMIMP string above?

Reply:
Yes